fury/actor.py

```diff
@@ -754,6 +754,95 @@ def callback(_caller, _event, calldata=None):
     return actor
 
 
+def dashed_line(start_pos, end_pos, num_lines=10, line_color=[1.0, 0.0, 0.0], line_fraction=0.5):
+     """Create an actor for one or more dashed lines.
+
+    Parameters
+    ------------
+    start_pos :  list (1, 3)
+    end_pos :  list (1, 3)
```

Comment:
to be consistent with the rest of the code, it should be a `np.array`. For an example, you can look at other actors